### PR TITLE
Improving the Sup Swicthover test

### PR DESCRIPTION
**Pull request by rohit-rp**
- Adding continious traffic test over LACP interfaces
- Adding negative and corner cases
- Made some Improving to the readme overall

### CODE REVIEW

File: feature/gnoi/system/tests/supervisor_switchover_test/README.md [hunk 1]
```diff
@@ -2,36 +2,132 @@
...
   /components/component/state/last-switchover-reason/details:
     platform_type: [ "CONTROLLER_CARD" ]
+  /components/component/state/redundant-role:
+    platform_type: [ "CONTROLLER_CARD" ]
+  /components/component/state/oper-status:
+    platform_type: [ "CONTROLLER_CARD" ]
+  
+  ## Interface and LACP State Paths ##
+  /interfaces/interface/state/oper-status:
+  /lacp/interfaces/interface/state/oper-status:
```
[the remark this answers is not in the document]
Comment (rohit-rp) [reply feature/gnoi/system/tests/supervisor_switchover_test/README.md]: Fixed

File: feature/gnoi/system/tests/supervisor_switchover_test/README.md [hunk 2]
```diff
@@ -2,36 +2,132 @@
...
+```
+
+### gNOI-3.3.1: Supervisor Switchover and Recovery Validation
+
+*   **Step 1:** Issue `gnoi.SwitchControlProcessor` to the chassis.
+*   **Step 2:** Validate the switchover was successful:
+    *   Verify the standby RE/SUP becomes active by checking `/components/component/state/redundant-role` transitions to `PRIMARY`.
+    *   Verify the old active RE/SUP transitions to `STANDBY`.
+*   **Step 3:** Validate traffic and LACP state during and after the switchover:
+    *   Verify the LACP session does not flap and connected ports remain up (`/interfaces/interface/state/oper-status` and `/lacp/interfaces/interface/state/oper-status` are `UP`).
```
[the remark this answers is not in the document]
Comment (rohit-rp) [reply feature/gnoi/system/tests/supervisor_switchover_test/README.md]: Fixed

File: feature/gnoi/system/tests/supervisor_switchover_test/README.md [hunk 3]
```diff
@@ -2,36 +2,132 @@
...
+        {
+          "name": "Ethernet2",
+          "config": {
+            "name": "Ethernet2"
+          },
+          "openconfig-if-ethernet:ethernet": {
+            "openconfig-if-aggregate:config": {
+              "aggregate-id": "Port-Channel1"
+            }
+          }
```
[the remark this answers is not in the document]
Comment (rohit-rp) [reply feature/gnoi/system/tests/supervisor_switchover_test/README.md]: Fixed

File: feature/gnoi/system/tests/supervisor_switchover_test/README.md [hunk 4]
```diff
@@ -2,36 +2,132 @@
...
+        {
+          "name": "Ethernet1",
+          "config": {
+            "name": "Ethernet1"
+          },
+          "openconfig-if-ethernet:ethernet": {
+            "openconfig-if-aggregate:config": {
+              "aggregate-id": "Port-Channel1"
+            }
+          }
```
[the remark this answers is not in the document]
Comment (rohit-rp) [reply feature/gnoi/system/tests/supervisor_switchover_test/README.md]: Fixed

**Review by AmrNJ — APPROVED** [review]
LGTM.

**Review by ram-mac — APPROVED** [review]
